android/sdl_android/src/main/java/com/smartdevicelink/transport/TCPTransport.java

```diff
@@ -293,23 +293,23 @@ public TransportType getTransportType() {
      * @param message Message to log
      */
     protected void logInfo(String message) {
-        Log.i(getClass().getName(), message);
+        DebugTool.logInfo(getClass().getName(), message);

```

Comment:
I suggest changing the TAG in this class to hardcoded TAG value just for consistency